app/views/issue_templates/_template_pulldown.html.erb

```diff
@@ -10,7 +18,7 @@
 <optgroup label="<%=h @tracker.name %>">
   <%= options_for_template_pulldown(grouped_options) %>
 </optgroup>
-<% if is_triggered_by_status.blank? || (is_triggered_by_status.present? && is_triggered_by_status == 'false') %>
+<% if is_triggered_by.blank? || (is_triggered_by.present? && is_triggered_by == 'issue_status_id') %>
```

Comment:
[IMO] I think ``is_triggered_by``  condition should not be ``issue_status_id``, because in this situation, the load request doesn't  fire. 
Maybe ``issue_tracker_id``  is better.

```
<% if is_triggered_by.blank? || (is_triggered_by.present? && is_triggered_by == 'issue_tracker_id') %>
```

Reply:
@ishikawa999 Thanks for your pull request!
Could you please confirm the point that I commented above?
It seems load action does not run when changing to another tracker which has some templates. 

Exp. 
---
1. Bug tracker has a default template.
2. Feature tracker has no template.
3. Support tracker has a default template.
--- 

Just after open new issue, default template for bug tracker is loaded.
After that, when change tracker to suppor, request for IssueTenplateContollerr#load is not fired.

Reply:
@akiko-pusu san
Thank you for the review.
I confirmed that the fix you suggested solves the problem.